src/App.css

```diff
@@ -0,0 +1,190 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 20px;
+}
+
+.header {
+  list-style: none;
+  padding: 0 2rem;
+}
+
+.headerimg {
+  width: 60px;
+  height: 60px;
+}
+
+.links > li {
+  list-style-type: none;
+}
+
+.links > li > a {
+  color: #0290ff;
+  list-style: none;
+  text-decoration: none;
+}
+
+.links > li:nth-child(2)::after {
+  content: '|';
+  float: right;
+  width: 1px;
+  height: 20px;
+  margin-left: 10px;
+}
+
+.links li a:hover,
+.links li a.active {
+  color: #0290ff;
+  text-decoration: underline;
+}
+
+.hr {
+  border: 0.1px solid lightgray;
+}
+
+.main {
+  background-color: white !important;
+}
+
+.main > div {
+  margin-top: 15px;
+}
+
+.Rocket_container {
+  padding: 0 2.5rem;
+}
+
+.Rocket_container > div {
+  height: 250px;
+  text-align: left;
+}
+
+.rocket_img > img {
+  width: 250px;
+  height: 100%;
+}
+
+.hidden {
+  display: none;
+}
+
+.show {
+  background-color: #26a69a;
+  border: none;
+  border-radius: 4px;
+  color: #fff;
+  margin-right: 6px;
+  padding: 0.1rem;
+}
+
+.rocket_head {
+  margin-top: -0.1%;
+  color: #383e4a;
+  font-family: sans-serif;
+  font-weight: 600;
+}
+
+.rocket_text {
+  font-weight: 300;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 18px;
+  color: rgb(62, 62, 62);
+}
+
+.cancel-btn,
+.reserve-Btn {
+  background-color: #0290ff;
+  border: none;
+  padding: 0.5rem;
+  color: white;
+  cursor: pointer;
+  border-radius: 5px;
+}
+
```

Comment:
Kindly adjust the responsiveness on the rockets page and align all the rockets rendered. This is the display on 1510px screen

![image](https://user-images.githubusercontent.com/20554583/233558447-3d4d2278-9d8d-4a23-8271-11e12000cead.png)